app/views/partials/personal_key/_key.html.erb

```diff
@@ -7,10 +7,17 @@
     </div>
   </div>
   <p class="margin-bottom-105">
-    <%= t(
-          'users.personal_key.generated_on_html',
-          date: content_tag(:strong, I18n.l(Time.zone.today, format: '%B %d, %Y')),
-        ) %>
+    <% if defined? personal_key_generated_at %>
+      <%= t(
+            'users.personal_key.generated_on_html',
+            date: content_tag(:strong, I18n.l(personal_key_generated_at, format: '%B %d, %Y')),
```

Comment:
I'd had some concern if it'd be possible for `personal_key_generated_at` to be `nil` here, since it is a possible return value of the new method, and would cause an exception to be raised. I _think_ it can't possibly happen due to [the controller's `before_action`s](https://github.com/18F/identity-idp/blob/fd14459a6af21cd419c2397e34dd37b1dda5d4e8/app/controllers/reactivate_account_controller.rb#L5), but could consider doing a graceful fallback anyways?

```suggestion
    <% if local_assigns[:personal_key_generated_at].present? %>
      <%= t(
            'users.personal_key.generated_on_html',
            date: content_tag(:strong, I18n.l(personal_key_generated_at, format: '%B %d, %Y')),
```